docs/reference/frozen-indices.asciidoc

```diff
@@ -64,4 +64,25 @@ The default value for `pre_filter_shard_size` is `128` but it's recommended to s
 significant overhead associated with this pre-filter phase.
 ================================
 
+== Monitoring frozen indices
+
+Frozen indices are ordinary indices that use search throttling and a memory efficient shard implementation. In order to monitor / filter
+frozen indices in APIs like the <<cat-indices>> they must be filtered by their `throttled` property since frozen indices are not a core
+concept.
```

Comment:
On first read, i was little bit confused by `in order to` (as in you must do this) and `not a core concept` (as in it is not first class feature).  

How about : 
Frozen indices are ordinary indices that use search throttling and a memory efficient shard implementation. For API's like the <<cat-indices>> frozen indicies may identified by an index's `throttled` property (`sth`).